code/fundamentals.rb

```diff
@@ -7,9 +7,18 @@
 puts message
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
-
+my_favoite_animal = {
+  name: Joey,
+}
 # 2. Print "My favorite animal is <your-favorite-animal>" by printing a string with `my_favorite_animal` interpolated into it.
-
+puts "my favorite animal is #{my_favorite_animal}"
```

Comment:
👍 (this will be a problem based on the definition of `my_favorite_animal` in question 1)